src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -19,22 +16,17 @@ namespace GraphQL.Server.Transports.AspNetCore
     public class GraphQLHttpMiddleware<TSchema>
         where TSchema : ISchema
     {
-        private const string JsonContentType = "application/json";
-        private const string GraphQLContentType = "application/graphql";
-        private const string FormUrlEncodedContentType = "application/x-www-form-urlencoded";
+        private const string _docsUrl = "See: http://graphql.org/learn/serving-over-http/.";
```

Comment:
```suggestion
        private const string DOCS_URL = "See: http://graphql.org/learn/serving-over-http/.";
```

To align with .editorconfig which will be added in the near future. We have one in core repo.